plugins/cliunarchiverplugin/cliplugin.cpp

```diff
@@ -189,24 +187,23 @@ void CliPlugin::readJsonOutput()
     foreach (const QJsonValue& value, entries) {
         const QJsonObject currentEntry = value.toObject();
 
-        m_currentEntry.clear();
+        m_currentEntry->clearMetaData();
```

Comment:
This is the only place where `clearMetaData()` is used. If `m_currentEntry` were not a global variable, you wouldn't need it. So please make this variable local and drop `clearMetaData()` from `Archive::Entry`